<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWidget</class>
 <widget class="QWidget" name="mainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-color: rgb(231, 231, 231);</string>
  </property>
  <widget class="QWidget" name="titleWidget" native="true">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>900</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(234, 234, 234);
background-image: url(:/title);
border-color: rgb(218, 218, 218);</string>
   </property>
   <widget class="BtnModel" name="closeBtn">
    <property name="geometry">
     <rect>
      <x>868</x>
      <y>4</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="leftWidget" native="true">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>37</y>
     <width>181</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(243, 243, 243);</string>
   </property>
   <widget class="QLineEdit" name="searchLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/searchEdit);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="allBtn">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>50</y>
      <width>172</width>
      <height>35</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>191</width>
      <height>35</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>  全部</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="meetBtn">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>83</y>
      <width>172</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>  会议通知</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="policyBtn">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>116</y>
      <width>172</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>  政策纪律宣贯</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="personBtn">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>149</y>
      <width>172</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>  人事任免通知</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line1">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>50</y>
      <width>172</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line2">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>83</y>
      <width>172</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line3">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>116</y>
      <width>172</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line4">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>149</y>
      <width>172</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line5">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>184</y>
      <width>172</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>900</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <zorder>searchLineEdit</zorder>
   <zorder>meetBtn</zorder>
   <zorder>policyBtn</zorder>
   <zorder>personBtn</zorder>
   <zorder>allBtn</zorder>
   <zorder>line1</zorder>
   <zorder>line2</zorder>
   <zorder>line3</zorder>
   <zorder>line4</zorder>
   <zorder>line5</zorder>
   <zorder>line</zorder>
  </widget>
  <widget class="QWidget" name="rightWidget" native="true">
   <property name="geometry">
    <rect>
     <x>179</x>
     <y>37</y>
     <width>721</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>440</y>
      <width>721</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="pageHLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>1</number>
     </property>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>49</y>
      <width>721</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="viewLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
    </layout>
   </widget>
   <widget class="QWidget" name="infoWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>720</width>
      <height>49</height>
     </rect>
    </property>
    <widget class="QLabel" name="msgCntLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="readCheckBox">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>10</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>仅看未读</string>
     </property>
    </widget>
    <widget class="ButtonLabel" name="retLabel">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>120</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="Line" name="splitLabel">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>11</y>
       <width>20</width>
       <height>28</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BtnModel</class>
   <extends>QPushButton</extends>
   <header>btnmodel.h</header>
  </customwidget>
  <customwidget>
   <class>ButtonLabel</class>
   <extends>QLabel</extends>
   <header>buttonlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
